ColabTool/Shared/Post/index.jsx

```diff
@@ -1,20 +1,46 @@
 import React, { PropTypes } from 'react';
 import {
   Card,
+  LinkifiedText,
 } from '@bufferapp/components';
 import style from './style.css';
 import PostFooter from '../PostFooter';
 import PostHeader from '../PostHeader';
 import RetweetPanel from '../RetweetPanel';
 
-const renderRetweetPanel = (retweetProfile) => {
+const renderRetweetComment = (comment, links) => (
+  <LinkifiedText
```

Comment:
for two args might be better to pass them in as an object like this

```
({ comment, links }) => //do stuff;
```

Makes it easier to refactor in the future!